packages/design-system/src/components/Dialog/__snapshots__/Dialog.test.jsx.snap

```diff
@@ -131,19 +131,13 @@ exports[`Dialog renders react-aria-modal 1`] = `
                     >
                       <svg
                         aria-hidden="true"
-                        aria-labelledby="icon-1__title"
                         class="ds-c-icon ds-c-icon--close "
                         focusable="false"
                         id="icon-1"
                         role="img"
```

Comment:
```suggestion
```

I think we should remove `role="img"` when `aria-hidden` is declared, so as to avoid a role accidentally getting announced.